### DISCUSSION TITLE
How to run in fork

### DISCUSSION THREAD

**Discussion by keiyu-lab** [silevis/reactgrid#377] [Q&A]
I need to add new features to this gird, so I want to customize it by forking it.
How can I debug it after forking?
because I am encountering an  error message "cannot access reactgrid before initialization" when I run $npm start.
I am new so I much appreciate your help

**Comment by qiufeihong2018** [comment]
Do you use pnpm?

**Reply by keiyu-lab** [reply]
Thanks for replying.
No I am using npm.

**Reply by qiufeihong2018** [reply]
You can use pnpm install && pnpm start
